tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1723,6 +1723,48 @@ describe("Chart evaluation", () => {
     ).toBe("#REF");
   });
 });
+
+describe("Cumulated Data line chart", () => {
+  test.each([
+    {
+      dataSets: ["B2:D7"],
```

Comment:
if you just test the first dataset data then we could reduce the set to ["B2:B7"] I guess.

Reply:
Hello @rrahir I've implemented the modifications you suggested. The dataset has been adjusted to [B2:B8] in order to incorporate non-numeric values and to verify that the cumulative values remain unaffected by the presence of non-numeric values in between.